src/toil/test/sort/sortTest.py

```diff
@@ -21,6 +21,7 @@
 import subprocess
 
 # Python 3 compatibility imports
+import sys
```

Comment:
actually, pyflakes is giving an 'imported but unused' warning here, so maybe just remove it entirely?